pkg/cli/serve/serve.go

```diff
@@ -0,0 +1,168 @@
+package serve
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	"github.com/openshift/ocm-agent/hack"
+)
+
+type level logrus.Level
+
+func (l *level) String() string {
+	return logrus.Level(*l).String()
+}
+
+func (l *level) Set(value string) error {
+	lvl, err := logrus.ParseLevel(strings.TrimSpace(value))
+	if err == nil {
+		*l = level(lvl)
+	}
+	return err
+}
+
+func (l *level) Type() string {
```

Comment:
what does this function do? seems it is not being used.

Reply:
Hey Meng Bo! Nice catch.. I do remember this implementation was required somewhere while accepting CLI flags but maybe I was mistaken while testing something else.. Wonder why IDE didn't catch this.. Yup, removed this bit.. Checked the CLI again and seems to work well